doc/howto/network_bgp.md

```diff
@@ -4,20 +4,60 @@ discourse: 11567
 
-Peers can be defined on both `bridged` and `physical` managed networks. Additionally in the `bridged` case, a set of per-server configuration keys are also available to override the next-hop. When those aren't specified, the next-hop defaults to the address used for the BGP session.
+The BGP server feature can be used to allow a LXD server or cluster to directly use internal/external address space by getting the specific subnets or addresses routed to the correct host.
+This way, traffic can be forwarded to the target instance.
 
-The `physical` network case is used for `ovn` networks where the uplink network is the one holding the list of allowed subnets and the BGP configuration. Once that parent network is configured, children OVN networks will get their external subnets and addresses announced over BGP with the next-hop set to the OVN router address for the network in question.
-
-The addresses and networks currently being advertised are:
- - Network `ipv4.address` or `ipv6.address` subnets when the matching `nat` property isn't set to `true`
- - Network `ipv4.nat.address` and `ipv6.nat.address` when those are set
+The following addresses and networks are being advertised:
+ - Network `ipv4.address` or `ipv6.address` subnets (if the matching `nat` property isn't set to `true`)
+ - Network `ipv4.nat.address` or `ipv6.nat.address` subnets (if the matching `nat` property is set to `true`)
  - Instance NIC routes defined through `ipv4.routes.external` or `ipv6.routes.external`
 
-At this time, there isn't a way to only announce some specific routes/addresses to particular peers. Instead it's currently recommended to filter prefixes on the upstream routers.
+At this time, it is not possible to announce only some specific routes/addresses to particular peers.
+If you need this, filter prefixes on the upstream routers.
+
+## Configure the BGP server
+
+To configure LXD as a BGP server, set the following server configuration options (see {ref}`server`):
+
+- `core.bgp_address` - the IP address for the BGP server
+- `core.bgp_asn` - the {abbr}`ASN (Autonomous System Number)` for the local server
+- `core.bgp_routerid` - the unique identifier for the BGP server
+
+For example, set the following values:
+
+```bash
+lxc config set core.bgp_address=192.0.2.0:179
+lxc config set core.bgp_asn=65536
+lxc config set core.bgp_routerid=192.0.2.0:179
+```
+
+Once these configuration options are set, LXD starts listening for BGP sessions.
+
+### Configure next-hop (`bridge` only)
+
+For bridge networks, you can override the next-hop configuration.
+By default, the next-hop is set to the address used for the BGP session.
+
+To configure a different address, set `bgp.ipv4.nexthop` or `bgp.ipvg.nexthop`.
+
+### Configure BGP peers for OVN networks
+
+If you run an OVN network with an uplink network (`physical` or `bridged`), the uplink network is the one that holds the list of allowed subnets and the BGP configuration.
```

Comment:
This talked only about `physical` before, but it is also for `bridge`, isn't it?